scripts/meta_analysis.py

```diff
@@ -190,74 +191,63 @@ def __lt__(self, other):
                   or (self.chr < other.chr)
                )
 
-    def is_equal(self, other:'VariantData') -> bool:
-        """
-            Checks if this VariantData is the same variant (possibly different strand or ordering of alleles)
-            returns: true if the same false if not
+    def is_equal(self, other: 'VariantData', equalize: bool = False, flip_indels: bool = False) -> bool:
+        """Checks if two variants are equal
+
+        Checks if this VariantData is the same variant as given other variant (possibly different strand or ordering of alleles).
+        
+        Args:
+            other:
+                Variant as VariantData object to compare this variant to.
+            equalize:
+                If True changes this variant's alleles and beta accordingly.
+            flip_indels:
+                If True will try matching indels with flipping.
+
+        Returns:
+            True if the same or False if not the same variant.
         """
 
         if (self.chr == other.chr and self.pos == other.pos):
-            flip_ref =  flip_strand(other.ref)
-            flip_alt =  flip_strand(other.alt)
 
-            if self.ref== other.ref and self.alt == other.alt :
+            if self.ref == other.ref and self.alt == other.alt :
                 return True
+            
+            if not flip_indels and self.is_indel:
+                return False
+            
+            flip_ref = flip_strand(other.ref)
+            flip_alt = flip_strand(other.alt)
 
-            if is_symmetric( other.ref, other.alt ):
+            if is_symmetric(other.ref, other.alt):
                 ## never strandflip symmetrics. Assumed to be aligned.
                 if self.ref == other.ref and self.alt == other.alt:
                     return True
                 elif self.ref == other.alt and self.alt == other.ref:
+                    if equalize:
+                        self.beta = -1 * self.beta if self.beta is not None else None
```

Comment:
now you are flipping symmetrics which we don't  want to but assume that they are in correct order (which they are in our case btw when we do ref mapping)

Reply:
That's an obvious mistake on my part, shouldn't be there.

Reply:
Actually seems like it has been there since the dawn of times. Probably not much of an effect since we haven't caught it before. Anyways good to fix it now!

Reply:
ah yea so it seems

Reply:
so should return false if symmetric and not straight up matching.

Reply:
Ok got it.

Reply:
Resolved in https://github.com/FINNGEN/META_ANALYSIS/pull/46/commits/92f9c998519fc1c73928cca97d85c2cd72d6d0a6.